src/index.html

```diff
@@ -11,7 +11,154 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="http://localhost:3000/api"
+      method="post">
```

Comment:
Wrong `action` value, you can get it from the readme file

Reply:
Also, fix the indentation, using the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md#:~:text=%5BCODE-,STYLE,-%5D%20%2D%20If%20the%20HTML)
```suggestion
    <form
      action="http://localhost:3000/api"
      method="post"
    >
```